src/content/structured/get-started/vue.mdx

```diff
@@ -56,7 +56,7 @@ export default {
       template: {
         compilerOptions: {
           // treat all tags with a dash as custom elements
```

Comment:
should this comment be changed to reference only "ic" elements rather than all custom elements?